test/unit/presenters/publishing_api/worldwide_organisation_presenter_test.rb

```diff
@@ -35,7 +35,7 @@ def present(...)
       document_type: "worldwide_organisation",
       locale: "en",
       publishing_app: Whitehall::PublishingApp::WHITEHALL,
-      rendering_app: "whitehall-frontend",
+      rendering_app: "government-frontend",
```

Comment:
This should use the constant as well ideally!

Reply:
Ooh, good idea.  I've swapped this to use the constant.